src/unix/bsd/freebsdlike/freebsd.rs

```diff
@@ -4,7 +4,16 @@ pub type fsfilcnt_t = ::uint64_t;
 pub const PTHREAD_STACK_MIN: ::size_t = 2048;
 pub const KERN_PROC_PATHNAME: ::c_int = 12;
 pub const SIGSTKSZ: ::size_t = 34816;
+pub const SF_NODISKIO: ::c_int = 0x00000001;
+pub const SF_MNOWAIT: ::c_int = 0x00000002;
+pub const SF_SYNC: ::c_int = 0x00000004;
+pub const SF_NOCACHE: ::c_int =   0x00000010;
 
 extern {
     pub fn __error() -> *mut ::c_int;
 }
+
+#[inline]
+pub fn SF_FLAGS(rh: ::c_int, flags: ::c_int) -> ::c_int {
```

Comment:
Having never used sendfile before, much less on freebsd, is this macro necessary for the usefulness of the function? Macros are unfortunately harder to validate, and I'd probably prefer to leave it if out if possible. If it's needed for inclusion, though, should be fine!